autogpt_platform/backend/.env.default

```
@@ -281,6 +281,10 @@ PLATFORM_LINK_BASE_URL=http://localhost:3000/link
 # CoPilot chat-platform bridge (Discord/Telegram/Slack/Teams)
 # Uses FRONTEND_BASE_URL (above) for link confirmation pages.
 AUTOPILOT_BOT_DISCORD_TOKEN=
+# Set to true only after enabling the Server Members privileged intent for the
+# bot in the Discord developer portal; the bot can then @mention any member by
+# display name. Roles work without it.
+AUTOPILOT_BOT_DISCORD_MEMBERS_INTENT=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AUTOPILOT_BOT_DISCORD_MEMBERS_INTENT key should go before the AUTOPILOT_BOT_DISCORD_TOKEN key

(UnorderedKey)